CENNZnet-API/Transaction-Codec.md

```diff
@@ -0,0 +1,175 @@
+# Extrinsic Codec Definition
+
+CENNZnet extrinsics are SCALE encoded payloads of the format
```

Comment:
This seems very cool but could you please explain how this is used in the API?

Reply:
Moved it to "CENNZnet developer section"